frontend/src/metabase/ui/components/inputs/SegmentedControl/SegmentedControl.styled.tsx

```diff
@@ -0,0 +1,54 @@
+import type {
+  MantineThemeOverride,
+  SegmentedControlStylesParams,
+} from "@mantine/core";
+import { getSize, rem } from "@mantine/core";
+
+const OPTION_PADDING = {
+  xs: rem(8),
+  sm: rem(8),
+  md: rem(8),
+};
+
+const OPTION_PADDING_FULL_WIDTH_V = {
+  xs: rem(8),
+  sm: rem(8),
+  md: rem(8),
+};
+
+const OPTION_PADDING_FULL_WIDTH_H = {
+  xs: rem(16),
```

Comment:
I'm not entirely sure we need a different padding in full width, mostly when used in full width the label should be centered and have more padding than specified here anyway. [I know they're different in the design](https://www.figma.com/file/6cHI5jaztfjDYvEXEhy9uE/Setup-flow-for-static%2Fpublic-embedding-%26-embedding-feature-discoverability?node-id=324%3A18628&mode=dev) (horizontal padding 8px vs 16px), but it might not be that apparent in actual application. Can you confirm with @vbenedetti if that's intended?

If we could use the same padding, it will simplify this override function a lot.

Reply:
fair enough.